src/main/java/Calculator.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Stack;
+
+public class Calculator {
+    public Calculator() {
+
+    }
+    private static Operator operator;
+    private static Postfix postfix;
```

Comment:
```suggestion
    private Operator operator;
    private Postfix postfix;
```

필드를 static 키워드를 통해 정적 멤버 필드로 선언하신 이유가 무엇일까요!?
static 필드를 최대한 사용자제 하면서 개발해보시는 것을 권장합니다.!